.github/workflows/docker_latest.yml

```diff
@@ -19,54 +25,88 @@ jobs:
       matrix:
         python-version: [3.9]
         node-version: [18.x]
+        image:
+          # We build two images, `grist-oss` and `grist`.
+          # See https://github.com/gristlabs/grist-core?tab=readme-ov-file#available-docker-images
+          - name: "grist-oss"
+            repo: "grist-core"
+          - name: "grist"
+            repo: "grist-ee"
+          # For now, we build it twice, with `grist-ee` being a
+          # backwards-compatible synoym for `grist`.
+          - name: "grist-ee"
+            repo: "grist-ee"
     steps:
       - name: Check out the repo
         uses: actions/checkout@v2
         with:
-          ref: latest_candidate
+          ref: ${{ inputs.latest_branch }}
+
+      - name: Check out the ext/ directory
+        if: matrix.image.name != 'grist-oss'
+        run: buildtools/checkout-ext-directory.sh ${{ matrix.image.repo }}
+
+
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
+
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+
       - name: Prepare image but do not push it yet
         uses: docker/build-push-action@v2
         with:
           context: .
           load: true
-          tags: ${{ github.repository_owner }}/grist:latest
+          tags: ${{ github.repository_owner }}/${{ matrix.image.name }}:latest
```

Comment:
I would like to propose, out of an abundance of caution, that we land this with an alternative tag, like `experimental`. The workflow looks good, but ending up with a `gristlabs/grist` image that acts enterprise-y would be very unsettling for users so I'd like to do this in two stages. Land this change but with an `experimental` tag. Test the generated images carefully. Then land a small change updating the tag. What do you think?

Reply:
Sounds reasonable.

Should I then create two extra workflows, by copying these two modified yaml files, and use that to build the new images with the new tag, while we keep the current workflows with the original, unmodified yaml files with the usual tags?

Or should we only temporarily pause the `latest` tag build for a short while until we can make sure that the `experimental` tag is working?

Reply:
After some discussion, we agree to just pause the `latest` build while we test that the `experimental` tag is working. I've made this change.